awslimits/support.py

```diff
@@ -3,12 +3,14 @@
 from boto3.dynamodb.conditions import Key, Attr
 from collections import namedtuple
 import dateutil.parser
+import os
 
 from dynamo_helpers import create_or_get_table
 
 TICKETS_TABLE_NAME = 'awslimits_tickets'
 LIMITS_TABLE_NAME = 'awslimits_limits'
 NAME_SEPARATOR = " :: "
+LIMIT_ALERT_PERCENTAGE = os.environ.get('LIMIT_ALERT_PERCENTAGE')
```

Comment:
I use 75% for warning and 90% for critical in all alerts I create. Since it's an env var, 90 as the default sounds fine to me. (I agree with @spulec, you need to `int()` that).